README.md

```diff
@@ -1800,25 +1804,24 @@ Typically, early to mid-stage companies grant stock options, and they may be
 - ☝️ This is partly because ISOs
   [can make it harder](http://thestartuplawblog.com/the-problem-with-immediately-exercisable-isos/)
   to meet the long-term capital gains holding period.
-  Many people expect early exercise together with an 83(b) election will help them hold the
-  stock longer, to qualify for long-term capital gains.
-  While this is true for NSOs, there is a murky part of the rules on ISOs that implies that
-  even with an 83(b) election, the capital gains holding period does not begin until the
-  shares actually vest.
-  So, if you want to immediately exercise an option and file a Section 83(b) election, and
-  you might have liquidity soon, it’s better if you can have it be an NSO.
+  Many people expect early exercise, together with an 83(b) election, will help them hold
+  the stock long enough to qualify for long-term capital gains.
+  While this is true for NSOs, a murky part of the rules on ISOs states that even with an
+  83(b) election, the capital gains holding period does not begin until the shares actually
+  vest. So if you want to immediately exercise an option and file a Section 83(b) election,
+  and you might have liquidity soon, it’s better if you can do so with NSOs.
```

Comment:
"if you can" -> "for those who can"

(was misleading in original)

Reply:
Fixed.